girder/utility/config.py

```diff
@@ -40,15 +40,15 @@ def _loadConfigsByPrecedent():
     """
     Load configuration in reverse order of precedent.
     """
-    configPaths = []
-    configPaths.append(
-        os.path.join(PACKAGE_DIR, 'conf', 'girder.dist.cfg'))
-    configPaths.append(
-        os.path.join(PACKAGE_DIR, 'conf', 'girder.local.cfg'))
-    configPaths.append(
-        os.path.join('/etc', 'girder.cfg'))
-    configPaths.append(
-        os.path.join(os.path.expanduser('~'), '.girder', 'girder.cfg'))
+    configPaths = [os.path.join(PACKAGE_DIR, 'conf', 'girder.dist.cfg')]
+
+    if 'GIRDER_TEST_DB' not in os.environ:
```

Comment:
I noticed that the test execution was being affected by how I had my girder.local.cfg set up, which is not desirable, so I went ahead and fixed that issue here.